Source/APIClient/APIClient.swift

```diff
@@ -2,16 +2,32 @@ import Alamofire
 
 final class APIClient {
 
+    enum HeaderType {
+        case chainpointJson
+        
+        func value() -> HTTPHeaders {
```

Comment:
I would rather extend the `BlockchainRequest` protocol with this header than have it on the `APIClient`